app/src/main/java/de/danoeh/antennapod/menuhandler/FeedItemMenuHandler.java

```diff
@@ -135,6 +143,40 @@ public static boolean onPrepareMenu(Menu menu, FeedItem selectedItem, int... exc
         return rc;
     }
 
+    public static boolean onMenuItemClicked(@NonNull Fragment fragment, int menuItemId,
```

Comment:
This class is for `FeedItem`s, aka episodes. I would create a second class that is for Feeds.